pkg/installation/utils.go

```diff
@@ -63,6 +77,54 @@ func (i *Installation) buildKymaInstaller(imageName string) error {
 	return nil
 }
 
+func (i *Installation) pushKymaInstaller(imageName string) error {
```

Comment:
Why is the image name passed as parameter if we already have it in `i.Options.CustomImage`?

Reply:
I just wanted to keep it as similar as possible to buildKymaInstaller function. There is no other reason. :)

Reply:
I changed it